examples_tests/20.Megatexture/main.cpp

```diff
@@ -489,33 +197,6 @@ core::smart_refctd_ptr<asset::ICPUSpecializedShader> createModifiedFragShader(co
     return fsNew;
 }
 
-constexpr uint32_t PAGETAB_SZ_LOG2 = 7u;
-constexpr uint32_t PAGETAB_LAYERS = 3u;
-constexpr uint32_t PAGETAB_MIP_LEVELS = 8u;
-constexpr uint32_t PAGE_SZ_LOG2 = 7u;
-constexpr uint32_t TILES_PER_DIM_LOG2 = 6u;
-constexpr uint32_t PHYS_ADDR_TEX_LAYERS = 3u;
-constexpr uint32_t PAGE_PADDING = 8u;
-enum E_TEX_PACKER
-{
-    ETP_8BIT,
-    ETP_24BIT,
-    ETP_32BIT,
-
-    ETP_COUNT
-};
-
-E_TEX_PACKER format2texPackerIndex(asset::E_FORMAT _fmt)
-{
-    switch (asset::getFormatClass(_fmt))
-    {
-    case asset::EFC_8_BIT: return ETP_8BIT;
-    case asset::EFC_24_BIT: return ETP_24BIT;
-    case asset::EFC_32_BIT: return ETP_32BIT;
-    default: return ETP_COUNT;
-    }
-}
-
 core::smart_refctd_ptr<asset::ICPUImage> createPoTPaddedSquareImageWithMipLevels(asset::ICPUImage* _img, asset::ISampler::E_TEXTURE_CLAMP _wrapu, asset::ISampler::E_TEXTURE_CLAMP _wrapv)
```

Comment:
this needs to become a `static` method of `ICPUVirtualTexture`

Reply:
still

Reply:
i see, dont add "still" to every unresolved comment XD if it's unresolved yet, it means i'll get it resolved later